src/coreclr/pal/src/thread/context.cpp

```diff
@@ -448,7 +474,7 @@ void CONTEXTToNativeContext(CONST CONTEXT *lpContext, native_context_t *native)
         // whether CONTEXT_FLOATING_POINT is set in the CONTEXT's flags.
         return;
     }
-#endif // HAVE_GREGSET_T || HAVE_GREGSET_T
+#endif // (HAVE_GREGSET_T || HAVE___GREGSET_T) && !defined(HOST_S390X)
```

Comment:
A nit - in the #endif comments, the convention we use is just `!HOST_S390X` instead of `!defined(HOST_S390X)`

Reply:
Ah, I see.  Updated.